io/pci/dlpar.py

```diff
@@ -79,14 +85,13 @@ def setUp(self):
                 self.log.warn("Incomplete hardware discovery!!. Refresh it")
                 self.cancel("Incomplete hardware discovery, skipping tests")
             self.lpar2_id = output.stdout_text[0]
-        self.pci_device = self.params.get("pci_device", default=None)
-        self.loc_code = pci.get_slot_from_sysfs(self.pci_device)
+        self.loc_code = pci.get_slot_from_sysfs(pci_device)
         self.num_of_dlpar = int(self.params.get("num_of_dlpar", default='1'))
```

Comment:
no need to call this for each pci ... move to setup

Reply:
as suggested the code is not needed to be called every time for each pci device, moved this line to setup() function.